sound/soc/sof/sof-priv.h

```diff
@@ -67,6 +67,9 @@ struct snd_sof_pdata;
  * and DSP device(s).
  */
 struct snd_sof_dsp_ops {
+
+	/* The following ops are mandatory */
```

Comment:
it's not very easy to figure out where the list stops. maybe for each op so that is can be searched whether it's optional or mandatory?